### DISCUSSION TITLE
将项目部署到服务器上，pm2启动无法使用

### DISCUSSION THREAD

**Discussion by PawNzZi** [1715173329/UnblockNeteaseMusic#739] [Q&A]
把项目部署到服务器后，如果用node启动app.js，ios代理后可以解锁，但是关闭ssh工具后服务就结束了，于是用pm2来守护进程，用pm2启动app.js后，播放音乐显示网络不给力，无法播放。
如果pm2启动不行，那么如何守护进程呢？

**Comment by pan93412** [comment]
pm2 應該是要可以運作的，大概是忘記把參數傳進去了……

**Reply by PawNzZi** [reply]
> 
参数传了的，程序能跑，但是无法播放。

**Comment by PawNzZi** [comment]
参数传了的，程序能跑，但是无法播放。

**Comment by 1715173329** [comment]
证书信任了吗？endpoint 设定了吗？

**Reply by PawNzZi** [reply]
> 证书信任了吗？endpoint 设定了吗？

信任了，也设置了
